packages/mermaid/src/docs.mts

```diff
@@ -68,6 +66,10 @@ const prettierConfig: prettier.Config = {
 
 let filesWereTransformed = false;
 
+const generateHeader = (file: string): string => {
+  return `# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT. Please edit the source file in [${file}](../${file}).`;
```

Comment:
The paths for `docs/developer-docs/development.md` should be `../../packages...` instead of just `../packages/...`

This can be fixed by the following (you'll also have to add an `import {relative} from "path";`:

```suggestion 
  // path from file in docs/* to repo root, e.g ../ or ../../ */
  const rootDirectory = relative(file, "packages/mermaid/src/docs");
  return `# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT. Please edit the source file in [${file}](${join(rootDirectory, file)}).`;
```